strictdoc/backend/sdoc/grammar/grammar.py

```diff
@@ -161,6 +162,36 @@
   )?
 ;
 
+DocumentView[noskipws]:
+  'DEFAULT_VIEW: ' default_view = SingleLineString '\n'
```

Comment:
Now that I see it implemented this way, I realize that we should rather move the `DEFAULT_VIEW` to the DocumentConfig as follows. It will make the meaning of this field consistent with the other fields because it is actually an option.

```
[DOCUMENT]
TITLE: Requirements
UID: SDOC_REQ
OPTIONS:
  REQUIREMENT_STYLE: Inline
  DEFAULT_VIEW: PRINT_VIEW
VIEWS:
...
```

Simply make the `DEFAULT_VIEW` to be the last field of the document config part. Also this makes sense when we get to editing the document options in the UI and the default view will be co-located with the other document options.

(This was not crystal clear for me in the beginning, sorry for making you to make this after we agreed on the format in our meeting).